traffic_ops/app/lib/TrafficOpsRoutes.pm

```diff
@@ -64,6 +64,12 @@ sub define {
 	# Traffic Stats Extension 1.4
 	$self->traffic_stats_routes( $r, $version );
 
+	# 1.5 Routes
```

Comment:
the formatting police are going to get you :)

Reply:
🚨 wee woo 🚨

Reply:
fixed. sorry about that